eos-reclaim-swap

```diff
@@ -55,6 +55,8 @@ class SwapRemover():
         '''
         dev = parted.getDevice(disk_path)
         disk = parted.newDisk(dev)
+        if disk.type == "msdos":
```

Comment:
Looks good to me, but maybe we should also double check if the flag is available?
```suggestion
        if disk.type == "msdos" and disk.isFlagAvailable(parted.DISK_CYLINDER_ALIGNMENT):
```